clients/cli/cmd/internal/pull.go

```diff
@@ -174,9 +174,16 @@ func (target *Target) DownloadAndWriteToFile(client *phrase.APIClient, localeFil
 		}
 	}
 
-	data, _ := ioutil.ReadAll(file)
-	file.Close()
-	os.Remove(file.Name())
+	var data []byte
+	if file != nil {
+		data, err = ioutil.ReadAll(file)
```

Comment:
> As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details.

If we're on 1.16 already let's adjust it to go with the recommendation.

Reply:
I thought about it but I did not want to do the go 1.16 upgrade in this PR.

Deploying the lambda function is still a manual process so it would mean we need force every deployer to upgrade to go1.16 or (would be better) migrate the deployment code to use docker.

Too much of a rabbit whole for this time.

Reply:
👍🏾